src/update-process-env.js

```diff
@@ -36,6 +33,10 @@ function updateProcessEnv (launchEnv) {
     for (let key in envToAssign) {
       if (!ENVIRONMENT_VARIABLES_TO_PRESERVE.has(key)) {
         process.env[key] = envToAssign[key]
+      } else {
+        if (!process.env[key] && envToAssign[key]) {
```

Comment:
Seems like this duplicated assignment could be eliminated by making the conditional:

```
if (!ENVIRONMENT_VARIABLES_TO_PRESERVE.has(key) || !process.env[key]) {
  process.env[key] = envToAssign[key]
}

```